### PR TITLE
Update gulpfile.js

### LINKED ISSUE #90: test

### PR DESCRIPTION
**Pull request by synicalsyntax**
Fixes #90

### CODE REVIEW

**Comment by zulipbot-test** [comment]
Hello @zulipbot-testing/server-development, @zulipbot-testing/server-hotkeys, @zulipbot-testing/server-misc members, this pull request was labeled with the "area: documentation (developer)", "area: hotkeys", "area: topics" labels, so you may want to check it out!

**Comment by zulipbot-test** [comment]
Hello @zulipbot-testing/server-development, @zulipbot-testing/server-hotkeys, @zulipbot-testing/server-misc members, this pull request was labeled with the "area: documentation (developer)", "area: hotkeys", "area: topics" labels, so you may want to check it out!

File: gulpfile.js [hunk 1]
```diff
@@ -60,3 +60,4 @@ gulp.task("coveralls", ["test"], function() {
 
 gulp.task("prepublish", ["nsp"]);
 gulp.task("default", ["static"]);
+// test
```
Comment (synicaltests) [inline gulpfile.js]: a

**Comment by zulipbot-test** [comment]
Heads up @synicalsyntax, we just merged some commits that conflict with the changes your made in this pull request! You can review this repository's [recent commits](https://github.com/zulipbot-testing/synicalsyntax/commits/master) to see where the conflicts occur. Please rebase your feature branch against the `upstream/master` branch and [resolve](https://zulip.readthedocs.io/en/latest/git/troubleshooting.html#recover-from-a-git-rebase-failure) your pull request's merge conflicts accordingly.

**Comment by zulipbot-test** [comment]
Hello @zulipbot-testing/server-development, @zulipbot-testing/server-hotkeys, @zulipbot-testing/server-misc members, this pull request was labeled with the "area: documentation (developer)", "area: hotkeys", "area: topics" labels, so you may want to check it out!

**Comment by zulipbot-test** [comment]
Hello @zulipbot-testing/server-development, @zulipbot-testing/server-hotkeys, @zulipbot-testing/server-misc members, this pull request was labeled with the "area: documentation (developer)", "area: hotkeys", "area: topics" labels, so you may want to check it out!

**Comment by zulipbot-test** [comment]
Hello @zulipbot-testing/server-development, @zulipbot-testing/server-hotkeys, @zulipbot-testing/server-misc members, this pull request was labeled with the "area: documentation (developer)", "area: hotkeys", "area: topics" labels, so you may want to check it out!

**Comment by zulipbot-test** [comment]
(unknown state)

**Comment by zulipbot-test** [comment]
Hello @zulipbot-testing/server-development, @zulipbot-testing/server-hotkeys, @zulipbot-testing/server-misc members, this pull request was labeled with the "area: documentation (developer)", "area: hotkeys", "area: topics" labels, so you may want to check it out!

**Comment by zulipbot-test** [comment]
Hello @zulipbot-testing/server-development, @zulipbot-testing/server-hotkeys, @zulipbot-testing/server-misc members, this pull request was labeled with the "area: documentation (developer)", "area: hotkeys", "area: topics" labels, so you may want to check it out!

**Comment by zulipbot-test** [comment]
Hello @zulipbot-testing/server-development, @zulipbot-testing/server-hotkeys, @zulipbot-testing/server-misc members, this pull request was labeled with the "area: documentation (developer)", "area: hotkeys", "area: topics" labels, so you may want to check it out!

**Comment by zulipbot-test** [comment]
Hello @zulipbot-testing/server-development, @zulipbot-testing/server-hotkeys, @zulipbot-testing/server-misc members, this pull request was labeled with the "area: documentation (developer)", "area: hotkeys", "area: topics" labels, so you may want to check it out!

**Pull request closed without merging**